GDJS/Runtime/inputmanager.js

```diff
@@ -42,21 +42,65 @@ gdjs.InputManager.MOUSE_RIGHT_BUTTON = 1;
 gdjs.InputManager.MOUSE_MIDDLE_BUTTON = 2;
 
 /**
- * Should be called whenever a key is pressed
+ * Holds the keyCodes of the keys which only have left / right
+ * variants and should thus default to their left variant values
+ * if necessary.
+ *
+ * @constant {number}
+ */
+gdjs.InputManager.DEFAULT_LEFT_VARIANT_KEYS = [16, 17, 18, 91];
+
+/**
+ * Returns the location aware keyCode.
+ *
+ * @private
+ */
+gdjs.InputManager.prototype._getLocationAwareKeyCode = function(
+  keyCode,
+  location
+) {
+  if (location) {
+    // If it is a numpad number, do not modify it.
+    if (96 <= keyCode && keyCode <= 105) {
+      return keyCode;
+    }
+
+    return keyCode + 1000 * location;
+  }
+
+  if (gdjs.InputManager.DEFAULT_LEFT_VARIANT_KEYS.includes(keyCode)) {
```

Comment:
Let's use indexOf to preserve IE11 compatibility as much as possible.

Reply:
Done!